### ISSUE TITLE
Validate links in generated documentation

### ISSUE THREAD

**Issue by valery1707** [rclone/rclone#4027]
As supposed in https://github.com/rclone/rclone/pull/4017#issuecomment-593073774 the build process should validate links in generated documentation (mostly `MANUAL.md` because of all other file generated from this one).

Right after executing the script `make_manual.py` or even inside this script we should validate that generated documentation doesn't contain any relative links.

**Comment by ncw** [comment]
Do you want to have a go at fixing this @valery1707 ?

**Comment by valery1707** [comment]
Yes, I want, but I don't sure about the mechanism for such validation.

Is it correct to add commands into the goal `MANUAL.md` inside `Makefile`, or is it better to modify `make_manual.py` itself?

**Comment by ncw** [comment]
> Yes, I want, but I don't sure about the mechanism for such validation.
> 
> Is it correct to add commands into the goal `MANUAL.md` inside `Makefile`, or is it better to modify `make_manual.py` itself?

Either would be fine I think. If you feel more comfortable editing `make_manual.py`  then you can put the check there.
